cvxpylayers/torch/cvxpylayer.py

```diff
@@ -106,12 +106,13 @@ def __init__(self, problem, parameters, variables, gp=False):
                     raise ValueError("An initial value for each parameter is "
                                      "required when gp=True.")
             data, solving_chain, _ = problem.get_problem_data(
-                solver=cp.SCS, gp=True)
+                solver=cp.SCS, gp=True, solver_opts={'use_quad_obj': False})
```

Comment:
it's just ``use_quad_obj = False``, no solver_opts.

Reply:
Are you sure? Setting it in `solver_opts` fixes the linked issue and setting `use_quad_obj=False` raises:

```python
  File ~/repos/cvxpylayers/cvxpylayers/torch/cvxpylayer.py:115 in __init__
    data, _, _ = problem.get_problem_data(
TypeError: get_problem_data() got an unexpected keyword argument 'use_quad_obj'

> /private/home/bda/repos/cvxpylayers/cvxpylayers/torch/cvxpylayer.py(115)__init__()
    113             self.param_ids = [p.id for p in self.compiler.parameters]
    114         else:
--> 115             data, _, _ = problem.get_problem_data(
    116                 # solver=cp.SCS, solver_opts={'use_quad_obj': False})
    117                 solver=cp.SCS, use_quad_obj=False)
```

Reply:
Ah sorry, ``get_problem_data`` is different than solve. The ``solver_opts`` way is the better way to do it, but it's not in ``solve`` for legacy reasons.